xlive/Globals.cpp

```diff
@@ -25,9 +25,9 @@ Players* players = new Players();
 bool microphoneEnabled = true;
 std::unordered_map<XUID, BOOL> xuidIsTalkingMap;
 std::unordered_map<XUID, anyID> xuidToTSid;
-IN_ADDR clientMachineAddress = {};
-TSClient* client = NULL;
-TSServer* server = NULL;
+IN_ADDR clientMachineAddress = { };
+TSClient* TS_client = NULL;
```

Comment:
why...this naming is completely different with naming of every other variable in this file?

Reply:
im sure the new naming is better than "server" and "client"